### ISSUE TITLE
Tailwindcss warnings after upgrade to 1.4.0

### ISSUE THREAD

**Issue by Dodobibi** [mrcaidev/tailwindcss-radix-colors#22]
After upgrading to 1.4.0, since you parse all the colors of tailwindcss, theses warnings appears :
 
 ```
 WARN                                                                                                                      12:24:55 PM


 WARN  warn - As of Tailwind CSS v2.2, lightBlue has been renamed to sky.                                                  12:24:55 PM


 WARN  warn - Update your configuration file to silence this warning.                                                      12:24:55 PM


 WARN                                                                                                                      12:24:55 PM


 WARN  warn - As of Tailwind CSS v3.0, warmGray has been renamed to stone.                                                 12:24:55 PM


 WARN  warn - Update your configuration file to silence this warning.                                                      12:24:55 PM


 WARN                                                                                                                      12:24:55 PM


 WARN  warn - As of Tailwind CSS v3.0, trueGray has been renamed to neutral.                                               12:24:55 PM


 WARN  warn - Update your configuration file to silence this warning.                                                      12:24:55 PM


 WARN                                                                                                                      12:24:55 PM


 WARN  warn - As of Tailwind CSS v3.0, coolGray has been renamed to gray.                                                  12:24:55 PM


 WARN  warn - Update your configuration file to silence this warning.                                                      12:24:55 PM


 WARN                                                                                                                      12:24:55 PM


 WARN  warn - As of Tailwind CSS v3.0, blueGray has been renamed to slate.                                                 12:24:55 PM


 WARN  warn - Update your configuration file to silence this warning. 
 ```

**Comment by mrcaidev** [comment]
This is a known issue. It seems necessary to hard-code Tailwind colors into the codebase in order to suppress these warnings. These warnings won't do any harm.

Anyway I will try to figure out a neat way to work around with this.

**Comment by mrcaidev** [comment]
Never mind. Fixed it in v1.4.1.
